.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Build and install wheel
+      run: |
+        python -m pip install --upgrade pip
+        pip install virtualenv
+        test/test_wheel_install.sh
+    # TODO: Add lint and test steps once we actually pass these tests.
+    # - name: Lint
```

Comment:
daf_butler has a lint.yaml workflow that you can copy directly. Having distinct workflows for distinct operations seems to be a good way of doing things since they all run in parallel.

Reply:
A neat thing is that jobs within a workflow will run in parallel too, unless there are dependencies between them. Github Actions is pretty great.

However... I still agree with Tim here, as it'd be super nice to be able to drop the same `lint.yaml` file into all of our packages, rather than maintaining special-snowflake workflows for each one.

Reply:
That's good with me, but I'd like to do that separately. DM-25199 blocks [DM-25104](https://jira.lsstcorp.org/browse/DM-25104), which blocks [DM-24645](https://jira.lsstcorp.org/browse/DM-24645), which blocks [DM-25226](https://jira.lsstcorp.org/browse/DM-25226), which has a real due date of 10 days. That's a deep chain of blockers! Adding linting is important but we should unblock the other work, and then add linting in parallel.